### PR TITLE
Added Share experiences file

### PR DESCRIPTION
**Pull request by anoura12**
Here, I added the file and changed the views accordingly. Experiences can also be entered in the page and will be saved in the local database. Right now, only the tags for the 1st and 3rd option work i.e. public and research.

### CODE REVIEW

**Comment by jhlink** [comment]
I'm sorry that I'm so late to review this PR!!

Reviewing now!

**Comment by jhlink** [comment]
Stellar job, @anoura12 !! 
You're very close!!

Could you add the remaining navbar items per issue #16 ? I've just updated it. So sorry for the tardiness!

File: server/apps/main/templates/main/share_experiences.html [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+                <a class="nav-link text-dark" href="{% url 'main:overview' %}">Home</a>
+              </li>
+              <li class="nav-item">
+                <a class="nav-link text-dark" href="#">Share</a>
+              </li>
+              <li class="nav-item">
+                <a class="nav-link text-dark" href="{% url 'main:list' %}">My Stories</a>
+              </li>
+              <li class="nav-item">
+                <a class="nav-link text-dark" href="#">Logout</a>
```
Comment (jhlink) [inline server/apps/main/templates/main/share_experiences.html:80]: Just need to add the missing links per the ticket #16 and this is good to merge!

**Review by jhlink — CHANGES_REQUESTED** [review]
Just need to add a couple changes, and this is good to merge!

File: server/apps/main/templates/main/share_experiences.html (hunk 1 above)
Comment (anoura12) [reply server/apps/main/templates/main/share_experiences.html:83]: Yes sure! I've pushed the new changes now.

Comment (jhlink) [reply server/apps/main/templates/main/share_experiences.html:83]: Well done!

**Review by jhlink — APPROVED** [review]
Well done!

**Pull request merged**